README.md

```diff
@@ -18,6 +18,8 @@ This GitHub action lets you push an Actor to the Apify platform, e.g. after ever
 
 **version** (optional): Actor version number to which the files should be pushed. By default, it is taken from the ".actor/actor.json" file.
 
+**working-directory** (optional): The working directory where the Actor files are located. By default, it is the root of the repository.
```

Comment:
After recent experience with `apify push`, I'd emphasize that this should point to a directory that contains the `.actor` folder and not to the `.actor` folder itself...